azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -388,4 +388,10 @@ public Executor fetchExecutorByExecutionId(int execId) throws ExecutorManagerExc
   public void unassignExecutor(int executionId) throws ExecutorManagerException {
     executionExecutorMapping.remove(executionId);
   }
+
+  @Override
+  public List<ExecutableFlow> fetchRecentlyFinishedFlows(long lifeTimeMs)
+      throws ExecutorManagerException {
+    return new ArrayList<>();
```

Comment:
Yes, from the return type it is able to figure out. If you explicitly put the type ArrayList<ExecutableFlow>, intellij will suggest you to remove it.